### DISCUSSION TITLE
Notification Bar Color does not match to the theme (on every app restart on Android)

### DISCUSSION THREAD

**Discussion by xthrk** [standardnotes/forum#2884] [General]
**Describe the bug**
The color of the Notification Bar does not match the theme color after every restart of the app on Android. Once I change the theme in Appearance, the color of the Notification Bar is also adjusted. But after every app restart this error occurs again.

**Expected behavior**
Notification bar color is matched to theme color

**Screenshots**
Notification Bar Color not matching to theme:
![grafik](https://user-images.githubusercontent.com/116081825/197982296-bce68f4e-55d1-4403-a7e8-67ed2de879ab.png)

Notification Bar Color after changing theme:
![grafik](https://user-images.githubusercontent.com/116081825/197982573-4c67c9e7-c966-4e8d-ab1a-bb21db2b1174.png)
![grafik](https://user-images.githubusercontent.com/116081825/197982731-9856f7b1-ee55-4a7a-938f-3afa0aa6ae77.png)

**Smartphone (please complete the following information):**
 - OS: Android 11
 - Version: 3.44.3

**Comment by effieeee** [comment]
Can replicate this as well, though since it also happens after an app restart, I'm wondering if it's somewhat distantly related to to what you mentioned [here](https://github.com/standardnotes/forum/issues/2232#issue-1423659748)? I'll keep this in mind!

**Comment by effieeee** [comment]
Looks like this has been fixed on v3.45.4, can I ask you to confirm as well? @crophit

**Comment by xthrk** [comment]
Hi @effieeee, thank you. I tested v3.45.4 (3002288) on an Android 11 and an Android 12 Device. On both devices, changing to correct the color of the Notification Bar works once a theme other than "Default" is selected (even after an app restart):
![grafik](https://user-images.githubusercontent.com/116081825/199570955-5ab20d76-4302-4651-b72a-f51edcea5531.png)

Unfortunately, the color of the "Default" theme still does not match. If you select a different theme and then switch back to "Default", the notification bar color is correct. But after restarting the app it looks like this again:
![grafik](https://user-images.githubusercontent.com/116081825/199571882-906621db-9651-4548-b932-9ea51f8e092a.png)

**Comment by effieeee** [comment]
I see 😞 okay, will keep this open then!
